cfgov/cfgov/settings/base.py

```diff
@@ -471,6 +471,11 @@
     'diffs': 'regcore.db.django_models.DMDiffs',
 }
 
+# Make regulations-site use Python to call regulations-core.
+# https://github.com/cfpb/regulations-site/pull/845
+if 'regcore' in INSTALLED_APPS and 'regulations' in INSTALLED_APPS:
+    EREGS_REGCORE_URLS = 'regcore.urls'
```

Comment:
Too late to this party for the merge, but this gives me such warmfuzzies on a very cold day. :heart: